aiohttp/web_urldispatcher.py

```diff
@@ -145,7 +145,7 @@ class StaticRoute(Route):
 
     def __init__(self, name, prefix, directory, *,
                  expect_handler=None, chunk_size=256*1024,
-                 response_factory=None):
+                 response_factory=None, sendfile_fallback=False):
```

Comment:
True, but it's useful to force the use of the fallback sendfile in the tests. That's the primary reason why the option exists.
But fallback is done automatically if `sendfile_fallback=False`. This should really be named `force_sendfile_fallback`, but that's too long.